finalproject/designreview1.md

```diff
@@ -0,0 +1,18 @@
+# Design Review 1
+
+You will have **8 minutes** to present the following information:
+
+- Elevator pitch - A brief (30-45 second) pitch of your project idea.
+What is it? Why is it exciting?
+
+- In-depth project description including:
```

Comment:
Did we really want the project description to be "in-depth" at this phase? Maybe "high level"/"big picture" instead?